Classes/Controller/Backend/Search/IndexQueueModuleController.php

```diff
@@ -147,21 +167,10 @@ public function initializeIndexQueueAction()
                 FlashMessage::OK
             );
 
-            $this->addFlashMessagesByQueueIdentifier('solr.queue.initializer');
+            //$this->addFlashMessagesByQueueIdentifier('solr.queue.initializer');
```

Comment:
I think in addition we should think about how to avoid the traits, e.g. by moving the code into a shared abstract class or a service that can be used as object. Technically traits are the same as copy & paste.

I think with merging this we have a better useable and implemented backend!